### DISCUSSION TITLE
plan for mapbox-gl in the coming few days

### DISCUSSION THREAD

**Discussion by snickell** [maps-gl/maps-gl#17] [General]
🤙🏽 everyone, I think first priority is:
- renaming human visible text to avoid mapbox trademark infringement
- encouraging people other than me to take over 🤣
- release a highly compatible build of maps-gl v1.13 to npm, with clear instructions on using it
- figure out how upstream bindings (react, vue, etc) can easily accomodate potentially multiple things that fulfill the same peer-dependency

**Comment by lseelenbinder** [comment]
Hi @snickell! Thanks for all your work so far.

We're trying to get consensus on which project to use going forward (including finding good governance, organization, etc.). Could you email me at luke at stadiamaps.com so I can get you added to the various conversations?
